src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/IdentifierValidator.cs

```diff
@@ -26,6 +26,8 @@ public static class IdentifierValidator
 
 		private const string Identifier = IdentifierStartCharacter + "(" + IdentifierPartCharacter + ")";
 
+		static Regex IsValidIdentifierRegex = new Regex ($"^[{IdentifierStartCharacter}][{IdentifierPartCharacter}]*$");
```

Comment:
Presumably this will be called a bunch, probably worth adding `RegexOptions.Compiled`?